src/test/java/org/vandeseer/easytable/util/PdfUtilTest.java

```diff
@@ -11,13 +11,27 @@
 
 public class PdfUtilTest {
 
+
+    /**
+     * The delta that is still acceptable in float comparisons.
+     */
+    private static final double EPSILON = 0.0001;
```

Comment:
The corresponding constant is public in `PdfUtil` – so no need to duplicate it ;)